_rules/meta-viewport-b4f0c3.md

```diff
@@ -22,7 +22,7 @@ acknowledgments:
 
 ## Applicability
 
-This rule applies to each `content` attribute on a `meta` element with a `name` [attribute value][] of `viewport`.
+This rule applies to each `content` attribute on a `meta` element with a `name` [attribute value][] of `viewport`. In addition, when the `content` attribute value is mapped to a list of property/value pairs in a user-agent specific manner, one of the properties is either `user-scalable` or `maximum-scale`.
```

Comment:
I'm quite reluctant in using "in a user-agent specific manner" since it sounds pretty ambiguous if user agents don't all do it the same way…
CSS is listing [this algorithm](https://drafts.csswg.org/css-device-adapt/#parsing-algorithm) for parsing it, which seems to be the iOS/Safari one. But also sounds pretty OK given what's normally in that list. At least, saying that the attribute is parsed that way would remove the ambiguity.